### PR TITLE
Switch to JSONNET config.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

File: .skiff/kube.yaml [hunk 1]
```diff
@@ -1,99 +0,0 @@
...
-kind: Deployment
-metadata:
-  labels:
-    app: lm-explorer
-  name: lm-explorer
-  namespace: lm-explorer
-spec:
-  replicas: 1
-  strategy:
-    rollingUpdate:
```
Comment (schmmd) [inline .skiff/kube.yaml:41]: @codeviking is this something I should preserve?

Comment (codeviking) [reply .skiff/kube.yaml:41]: Yes, I think so.

I believe so.  It should prevent downtime, as it's effectively a blue / green deploy.

File: .skiff/webapp.jsonnet [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                name: fullyQualifiedName,
+                namespace: namespaceName,
+                labels: labels
+            },
+            spec: {
+                containers: [
+                    {
+                        name: config.appName,
+                        image: image,
+                        args: [ 'server/start.py', '--prod' ],
```
Comment (codeviking) [inline .skiff/webapp.jsonnet]: The previous [manifest](https://github.com/allenai/lm-explorer/pull/11/files#diff-c13170b96a08fcaf0f67fb4566b08b61L56) didn't provide a command, so I think you'll need to remove this.

File: .skiff/webapp.jsonnet [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                        resources: {
+                            requests: {
+                                // Our machines currently have 2 vCPUs, so this
+                                // will allow 4 apps to run per machine
+                                cpu: '1',
+                                // Each machine has 13 GB of RAM. We target 4
+                                // apps per machine, so we reserve 3 GB of RAM
+                                // for each (whether they use it our not).
+                                memory: '6Gi'
+                            }
```
Comment (codeviking) [inline .skiff/webapp.jsonnet:200]: Should we try using less resources?

**Comment by codeviking** [comment]
@schmmd Pretty sure at the moment this will break, due to the `args` block that I think you can omit.

I'd love to see us try using less resources. I doubt this needs a single, dedicated CPU at all times :).

Memory might have to stay the same -- I'd ask @joelgrus 'bout that.

File: .skiff/webapp.jsonnet (hunk 2 above)
Comment (schmmd) [reply .skiff/webapp.jsonnet]: Thanks--I had thought I'd removed this.

File: .skiff/webapp.jsonnet (hunk 3 above)
Comment (schmmd) [reply .skiff/webapp.jsonnet:200]: Joel just upped this today out of need.  So probably not.

**Comment by joelgrus** [comment]
Screenshot from 2019-05-20 15-50-11

now you know as much as I do

**Comment by joelgrus** [comment]
looks like it should be ok with ~0.4 CPU + 5GB ?

**Comment by codeviking** [comment]
0.4 CPU and 5 GB sgtm!

**Comment by schmmd** [comment]
@joelgrus cool!  I would change those requests in a separate PR though.  I'd like this to simply be changing the configuration language.

**Comment by schmmd** [comment]
@joelgrus once you approve I will test these changes before merging.

**Review by joelgrus — APPROVED** [review]
you guys know way more about skiff configs than I do

**Comment by schmmd** [comment]
@joelgrus is it OK if I (test and then) merge tomorrow morning and troubleshoot any issues that come up (hopefully none)?

**Comment by joelgrus** [comment]
:+1:

**Pull request merged**